packages/next/compiled/terser-webpack-plugin/minify.js

```diff
@@ -2,7 +2,7 @@
 
 const {
   minify: terserMinify
-} = require("next/dist/compiled/terser");
+} = require('terser');
```

Comment:
I don't understand why this is no longer getting ncc'd, any ideas?

Reply:
Do you mean the file is missing when you leave this in?

Reply:
When I run `yarn ncc-compiled` this is the output, so it seems like `terser` isn't going through ncc

Reply:
See https://github.com/merceyz/next.js/pull/1 for the fix here.

Reply:
That didn't actually fix it but it pointed me to the place the issue was, cheers